src/FixedBasic/Redis.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+namespace Jitendra\PhpValve\FixedBasic;
+
+final class Redis extends Base
+{
+    const KEY_PREFIX = 'JPVFBR:';
+
+    /**
+     * @var \Predis\Client
+     */
+    protected $redis;
+
+    public function __construct(int $window, int $limit, \Predis\Client $redis = null)
+    {
+        parent::__construct($window, $limit);
+
+        $this->redis = $redis ?: new \Predis\Client;
+    }
+
+    public function attempt(string $resource, int $worth = 1): array
```

Comment:
why call this variable `worth`? any reference?

Reply:
@saurav1991 : Saw it somewhere on aws docs. For e.g. some resource are expensive than others so accessing them will cost you more.